tests/ha/ha_cluster_init.pm

```diff
@@ -29,8 +29,18 @@ sub run {
 
     # If we failed to initialize the cluster, trying again but in debug mode
     # Note: the default timeout need to be increase because it can takes time to join the cluster
-    if (script_run "ha-cluster-init -y -s $sbd_device $unicast_opt", $join_timeout) {
-        assert_script_run "crm -dR cluster init -y -s $sbd_device $unicast_opt", $join_timeout;
+    # Initialize the cluster with diskless or shared storage SBD
+    if (get_var('USE_DISKLESS_SBD')) {
+	record_info('SBD info', 'Using diskless SBD');
```

Comment:
travis is failing here. I guess you used a TAB instead of 8 spaces.

Reply:
thanks, I've just installed the good .perltidyrc.